src/frontend/Lex.mll

```diff
@@ -170,6 +170,8 @@ and real_token = parse
     { HOLE_NAME None }
   | "!"
     { BANG }
+  | "!?"
```

Comment:
I would like to use something different here. Maybe like `#viz` or something? idk.

Reply:
Sounds good! In prep for this I've bitten the bullet and implemented #281.